pom.xml

```diff
@@ -193,4 +193,15 @@
 			</plugins>
 		</pluginManagement>
 	</build>
+	<profiles>
```

Comment:
I read around that this is a solution to avoid too-strict checks from the Javadoc utility that comes with JDK 8. Is that correct?

It seems good to me, but my understanding of this is limited